codelists/actions.py

```diff
@@ -256,12 +256,15 @@ def create_version_with_codes(
     if not codes:
         raise ValueError("No codes")
     prev_clv = codelist.versions.order_by("id").last()
+    coding_system = prev_clv.codelist.coding_system_cls.get_by_release(
```

Comment:
We've already got `codelist`, so no need to get it from `prev_clv.codelist`.

Reply:
Ah, good point!